inst/odin/carehomes.R

```diff
@@ -545,32 +682,34 @@ initial(I_ICU_S_D_unconf[, , ]) <- 0
 initial(I_ICU_S_D_conf[, , ]) <- 0
 initial(I_ICU_D_unconf[, , ]) <- 0
 initial(I_ICU_D_conf[, , ]) <- 0
-initial(R_stepdown_R_unconf[, ]) <- 0
-initial(R_stepdown_R_conf[, ]) <- 0
-initial(R_stepdown_D_unconf[, ]) <- 0
-initial(R_stepdown_D_conf[, ]) <- 0
-initial(R_pre[, ]) <- 0
-initial(R_pos[, ]) <- 0
-initial(R_neg[]) <- 0
-initial(R[]) <- 0
+initial(R_stepdown_R_unconf[, , ]) <- 0
+initial(R_stepdown_R_conf[, , ]) <- 0
+initial(R_stepdown_D_unconf[, , ]) <- 0
+initial(R_stepdown_D_conf[, , ]) <- 0
+initial(R_pre[, , ]) <- 0
+initial(R_pos[, , ]) <- 0
+initial(R_neg[, ]) <- 0
+initial(R[, ]) <- 0
 initial(D_hosp[]) <- 0
 initial(D_comm[]) <- 0
-initial(PCR_pre[, ]) <- 0
-initial(PCR_pos[, ]) <- 0
-initial(PCR_neg[]) <- 0
+initial(PCR_pre[, , ]) <- 0
+initial(PCR_pos[, , ]) <- 0
+initial(PCR_neg[, ]) <- 0
 initial(cum_admit_conf) <- 0
 initial(cum_new_conf) <- 0
 initial(cum_admit_by_age[]) <- 0
 
 ## User defined parameters - default in parentheses:
 
-## Parameters of the S classes
+## Vaccination parameters
 rel_susceptibility[, ] <- user()
 dim(rel_susceptibility) <- user() # use length as provided by the user
 n_vacc_classes <- dim(rel_susceptibility, 2)
+rel_p_sympt[, ] <- user()
+dim(rel_p_sympt) <- user() # use length as provided by the user
```

Comment:
This should be

```
dim(rel_p_sympt) <- n_vacc_classes
```

I believe (and probably also with the similar quantity in #157)